sql/core/src/main/scala/org/apache/spark/sql/execution/command/SetCommand.scala

```diff
@@ -79,7 +79,7 @@ case class SetCommand(kv: Option[(String, Option[String])]) extends RunnableComm
     // Queries all key-value pairs that are set in the SQLConf of the sparkSession.
     case None =>
       val runFunc = (sparkSession: SparkSession) => {
-        sparkSession.conf.getAll.map { case (k, v) => Row(k, v) }.toSeq
+        sparkSession.conf.getAll.map { case (k, v) => Row(k, v) }.toSeq.sortBy(_.getString(0))
```

Comment:
Is it simpler to sort earlier?  `sparkSession.conf.getAll.toSeq.sorted.map { case (k, v) => Row(k, v) }`